cmd/kustomize/main.go

```
@@ -61,18 +62,28 @@ func New(testCtx *test.TestCommandContext, kustomizeCtx *KustomizeContext) *cobr
 			}
 
 			localConfigContent, err := testCtx.LocalConfig.GetLocalConfiguration()
-			if err != nil {
+			isBackendAvailable := testCtx.CliClient.IsBackendAvailable()
+
+			if err != nil && isBackendAvailable {
 				return err
 			}
 
-			evaluationPrerunData, err := testCtx.CliClient.RequestEvaluationPrerunData(localConfigContent.Token)
+			evaluationPrerunData := &cliClient.EvaluationPrerunDataResponse{}
 
+			if isBackendAvailable {
+				evaluationPrerunData, err = testCtx.CliClient.RequestEvaluationPrerunData(localConfigContent.Token)
```

it looks like the way isBackendAvailable is being used is always the same

Move the `isBackednAvailable` logic inside the clients function so that it will be exposed only to cliClient, that way when you request something and backend is down the cliClient function will return the empty value of what you request because that's what you're doing here anyway.

This will also require the offline parameter to be passed to the cliClient so it will know if it should return an error or an empty value